Blog/templates/Blog/post_detail.html

```diff
@@ -17,4 +17,7 @@ <h2>Comments</h2>
         </div>
         <p class="short-padding"></p>
     {% endfor %}
+    <div data-role="panel">
+        <a href="{{ post.get_absolute_url }}/add"><p>Add your own comment!</p></a>
```

Comment:
One should never hard code urls (`/add`). Reason: if the url pattern is changed, all the code has to be updated accordingly.

Instead, one should use the template tag [url](https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#url). This way the url patterns can change and the urls will be resolved automatically.